addons/purchase_stock/tests/test_purchase_order_suggest.py

```diff
@@ -14,9 +14,7 @@ class TestPurchaseOrderSuggest(PurchaseTestCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.delivery_type = cls.env['stock.picking.type'].browse(cls.picking_type_out)
-        cls.stock_location = cls.delivery_type.default_location_src_id
-        cls.customer_location = cls.delivery_type.default_location_dest_id
+        cls.delivery_type = cls.picking_type_out
```

Comment:
You can remove this line as well and update the definition of `delivery_type` in `_create_and_process_delivery_at_date`:
```python
delivery_type = warehouse.out_type_id if warehouse else self.picking_type_out
```

Reply:
Removed and changed, thanks! :+1: